routers/web/repo/pull.go

```diff
@@ -362,6 +362,9 @@ func setMergeTarget(ctx *context.Context, pull *issues_model.PullRequest) {
 // GetPullDiffStats get Pull Requests diff stats
 func GetPullDiffStats(ctx *context.Context) {
 	issue := checkPullInfo(ctx)
+	if issue == nil {
```

Comment:
Maybe move line 372->374 here?

Reply:
Perhaps we could add a `bool` return value to `checkPullInfo` to determine if the check passes?
```
func checkPullInfo(ctx *context.Context) (*issues_model.Issue, bool) {
```
```
	issue, ok := checkPullInfo(ctx)
	if !ok {
		return
	}
```

Reply:
> I don't think the readability of using `ctx.Written()` is very good ，Perhaps we could add a `bool` return value to `checkPullInfo` to determine if the check passes?
> 
> ```
> func checkPullInfo(ctx *context.Context) (*issues_model.Issue, bool) {
> ```
> 
> ```
> 	issue, ok := checkPullInfo(ctx)
> 	if !ok {
> 		return
> 	}
> ```

That's also OK.